maven-core/src/main/java/org/apache/maven/internal/impl/PropertiesAsMap.java

```diff
@@ -71,6 +71,8 @@ public Entry<String, String> next()
                         {
                             throw new NoSuchElementException();
                         }
+                        Entry<Object, Object> next = this.next;
```

Comment:
please use another name for local variable ... it is not clear which `next` is used

Reply:
Fixed